lib/migrations/2_deploy_organization.ts

```diff
@@ -1,21 +1,55 @@
+import Web3 = require("web3");
+import { computeGasLimit } from "../../gasLimits.js";
 import { DefaultSchemePermissions, SchemePermissions } from "../commonTypes";
 import { ConfigService } from "../configService";
 import { GetDefaultGenesisProtocolParameters } from "../wrappers/genesisProtocol";
-const computeGasLimit = require("../../gasLimits.js").computeGasLimit;
 
-/*
+/* tslint:disable:no-console */
+/* tslint:disable:max-line-length */
+
+interface FounderSpec {
+  /**
+   * Founders' address
+   */
+  address: string;
+  /**
+   * string | number token amount to be awarded to each founder, in GEN
+   */
+  tokens: string | number;
+  /**
+   * string | number reputation amount to be awarded to each founder,
+   * in units of the Genesis Reputation system.
+   */
+  reputation: string | number;
+}
 /**
  * Migration callback
  */
-export const arcJsDeployer = (web3, artifacts, deployer) => {
+export const arcJsDeployer = (web3: Web3, artifacts: any, deployer: any): void => {
 
   const network = ConfigService.get("network") || "ganache";
-  const founders = require("../../migrations/founders.json").founders[network];
+
+  const internalFoundersConfigLocation = "../../migrations/founders.json";
+  const foundersConfig = require(internalFoundersConfigLocation).founders;
+
+  const customFoundersConfigLocation = ConfigService.get("foundersConfigurationLocation");
+
+  if (internalFoundersConfigLocation !== customFoundersConfigLocation) {
+    console.log(`merging custom founders from ${customFoundersConfigLocation}`);
```

Comment:
seems like you should overwrite, not merge

Reply:
this does overwrite

Reply:
that would merge the founders from the foundersConfig with the customFoundersConfig. If you pass in a custom founders config why would you also want the default/internal founders?

Reply:
ok